### PR TITLE
Fix re-declaration issue of 'stack' method

### LINKED ISSUE #133: stack metod defined twice
The ' **stack** ' static method **defined twice** in tensorly/backend/pytorch_backend.py
`    @staticmethod
      def stack(arrays, axis=0):
           return torch.stack(arrays, dim=axis)
    `

### PR DESCRIPTION
**Pull request by Cor3Down**
Hi Mr @JeanKossaifi , I have fixed #133 by Removing second declaration of this method.

### CODE REVIEW

**Comment by JeanKossaifi** [comment]
Hi @Cor3Down,
Thanks for the PR.
In general, there is no need to open an issue with each PR, especially for minor changes like this. 
It is more common to condense related changes into one PR (e.g. #132 could have been merged with this as they box fix typos/minor issues in the pytorch backend, typos can be grouped in one PR, etc).
Merging this, thanks!

**Comment by Cor3Down** [comment]
Hi Dear @JeanKossaifi.
Thanks for your valuable tips.
